src/map_to_curve.rs

```diff
@@ -56,6 +56,35 @@ impl GroupWithMapToCurve for RistrettoPoint {
     }
 }
 
+#[cfg(feature = "p256")]
+impl GroupWithMapToCurve for p256_::ProjectivePoint {
+    const SUITE_ID: usize = 0x0003;
```

Comment:
I'm not sure where I got this from, but looking at suite ids in the OPAQUE draft, we aren't following the suggested ones anyway. Is `0x0003` alright? I guess `0x0002` could be reserved for decaf448.

Reply:
Ah, alright, I was looking at OPAQUE suite ids, which is totally different.